app/assets/stylesheets/pages/_chef_profiles.scss

```diff
@@ -0,0 +1,9 @@
+.card-preview {
```

Comment:
Could you give the card a different class name and put it within the components?